scala/src/main/scala/smile/clustering/k-modes/KModes.scala

```diff
@@ -0,0 +1,92 @@
+package smile.clustering.kmodes
```

Comment:
Don't create the package `kmodes`. The algorithm should be just in the `smile.clustering` package.